### PR TITLE
moje izmene

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
pozdrav

**Comment by ghost** [comment]
evo i kategorija na strani products

File: controller/Controller.php [hunk 1]
```diff
@@ -11,50 +11,53 @@ function test_input($data){
 
 class Controller{
 
-	
+
 	public function index(){
+		$dao=new DAO();
```
Comment (PedjaHC) [inline controller/Controller.php:16]: Ok, prava izmena, kreiranje dao objekta

**Review by PedjaHC — APPROVED** [review]
Prihvaćene promene - Žiletove promene

File: controller/Controller.php [hunk 2]
```diff
@@ -11,50 +11,53 @@ function test_input($data){
 
 class Controller{
 
-	
+
 	public function index(){
+		$dao=new DAO();
+		$categories=$dao->getCategoriesAndSubcategories();
```
Comment (PedjaHC) [inline controller/Controller.php:17]: Ok promena, stvarna, punjenje promenljive $categories

**Review by PedjaHC — APPROVED** [review]
Prihvaćene promene - Žiletove promene

File: controller/Controller.php [hunk 3]
```diff
@@ -11,50 +11,53 @@ function test_input($data){
 
 class Controller{
 
-	
+
 	public function index(){
+		$dao=new DAO();
+		$categories=$dao->getCategoriesAndSubcategories();
+		var_dump($categories);
```
Comment (PedjaHC) [inline controller/Controller.php:18]: ok promena, var dump

**Review by PedjaHC — APPROVED** [review]
Prihvaćene promene - Žiletove promene

File: controller/Controller.php (hunk 1 above)
Comment (PedjaHC) [inline controller/Controller.php:16]: ok promena

File: controller/Controller.php (hunk 2 above)
Comment (PedjaHC) [inline controller/Controller.php:17]: ok promena

File: controller/Controller.php (hunk 3 above)
Comment (PedjaHC) [inline controller/Controller.php:18]: Ok promena

File: controller/Controller.php [hunk 4]
```diff
@@ -110,59 +113,62 @@ public function login(){
 		$email=isset($_POST['email'])?$_POST['email']:"";
     	$password=isset($_POST['password'])?$_POST['password']:"";
         $user=isset($user)?$user:"";
-		
-        if(!isset($email)){       
+        $errors=array();
```
Comment (PedjaHC) [inline controller/Controller.php:116]: ok promena

File: controller/Controller.php [hunk 5]
```diff
@@ -110,59 +113,62 @@ public function login(){
 		$email=isset($_POST['email'])?$_POST['email']:"";
     	$password=isset($_POST['password'])?$_POST['password']:"";
         $user=isset($user)?$user:"";
-
```
Comment (PedjaHC) [inline controller/Controller.php:113]: ok promena

File: controller/Controller.php [hunk 6]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+    	if(count($errors)==0){
  			if (!isset($_SESSION)) {
-				session_start(); 
-			}	              				
-			$_SESSION['user']=serialize($user);			
-          	include 'index.php';                    
+				session_start();
+			}
+			$_SESSION['user']=serialize($user);
+				$dao=new DAO();
```
Comment (PedjaHC) [inline controller/Controller.php:145]: ok

File: controller/Controller.php [hunk 7]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+			  $categories=$dao->getCategoriesAndSubcategories();
```
Comment (PedjaHC) [inline controller/Controller.php:146]: ok

File: controller/Controller.php [hunk 8]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+			 //var_dump($categories);
```
Comment (PedjaHC) [inline controller/Controller.php:147]: ok

File: controller/Controller.php [hunk 9]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+          	include 'index.php';
```
Comment (PedjaHC) [inline controller/Controller.php:148]: ok

File: controller/Controller.php [hunk 10]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+    		$errors['password']='Wrong password';
     	}
 
-    	if(count($errors)==0){                
+    	if(count($errors)==0){
  			if (!isset($_SESSION)) {
-				session_start(); 
-			}	              				
-			$_SESSION['user']=serialize($user);			
-          	include 'index.php';
```
Comment (PedjaHC) [inline controller/Controller.php:142]: ok

File: controller/Controller.php [hunk 11]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
 		if (!isset($_SESSION)){
-			session_start(); 
-		}		
+			session_start();
+		}
 		session_unset();
 		session_destroy();
-		include 'login.php';	
+		include 'login.php';
+	}
```
Comment (PedjaHC) [inline controller/Controller.php:163]: ok

File: controller/Controller.php [hunk 12]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+  public function showProducts(){
```
Comment (PedjaHC) [inline controller/Controller.php:164]: ok

File: controller/Controller.php [hunk 13]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 $subId=isset($_GET['id_sub_category'])?$_GET['id_sub_category']:"";
```
Comment (PedjaHC) [inline controller/Controller.php:165]: ok

File: controller/Controller.php [hunk 14]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 //var_dump($subId);
```
Comment (PedjaHC) [inline controller/Controller.php:166]: ok

File: controller/Controller.php [hunk 15]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 $dao=new DAO();
```
Comment (PedjaHC) [inline controller/Controller.php:167]: ok

File: controller/Controller.php [hunk 16]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 $productsBySubId=$dao->getProductsBySubcategoryId($subId);
```
Comment (PedjaHC) [inline controller/Controller.php:168]: ok

File: controller/Controller.php [hunk 17]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 $categories=$dao->getCategoriesAndSubcategories();
```
Comment (PedjaHC) [inline controller/Controller.php:169]: ok

File: controller/Controller.php [hunk 18]
```diff
@@ -110,59 +113,62 @@ public function login(){
...
+		 include 'products.php';
```
Comment (PedjaHC) [inline controller/Controller.php:170]: ok

**Review by PedjaHC — APPROVED** [review]
Prihvaćene promene u conntroler.php - Žiletove promene

**Pull request merged**